src/buildblock/zoom.cxx

```diff
@@ -442,6 +445,57 @@ zoom_image(VoxelsOnCartesianGrid<float> &image_out,
 
   overlap_interpolate(image_out, temp2, zoom_z, z_offset);
 
+
+  /* The code provides for three possible rescaling options:
+   * preserving the image values: should be used when zooming an attenuation image
+   * preserving the image projectors: should be used when zooming an activity image
+   * preserving the image sum: DEFAULT*/
+
+  float scale_image = 1.F;
+
+    switch (zo)
+  {
+  case ZoomOptions::preserve_values:
+  {
+      std::cerr << "Zoom - Rescaling Factor: preserve values\n";
```

Comment:
no output to std::cerr. you could call `info("Zoom - Rescaling Factor: preserve values")` instead, but not sure that we need it